data/human/deep missions.txt

```diff
@@ -2752,3 +2752,1453 @@ mission "Stone of our Fathers 5"
+			
+	on complete
+		outfit "Tear of Skade"
+		conversation		
+			`Nifel is colder than you remember. The snow is deeper, and the ice harder. Despite this, the small recolonized settlement of Udainsakr appears to be thriving. Although it started with less than 50 Skadenga, thanks to the steady flow of pilgrims, the village population now numbers over a thousand.`
+			`	Hjlod finds you and gives you a small smile. "I am glad you came."`
+			`	She reaches into her pocket and pulls out a small wooden box, similar to the one you saw displayed at Asgard years ago. "I have been thinking," Hjlod says. "Maybe all this vas vill of Skade. Maybe you took us to Asgard to make us stronger, to make us better. It is bitter lesson, but I understand it now."`
+			` 	She holds out the box. "This is true of Skadenga. Only few ever found, and this I give to you. For saving my life, for bringing us back here, for making us stronger."`
+			scene "outfit/skadetear"
+			`	The box opens, displaying a small, blue glowing rock. "This is Tear of Skade. Alvays makes cold. Alvays. Useful on hot ship. Keep this vith you, and know Skadenga vith you too," Hjlod says, handing it to you.`	
+			`	Her eyes water, and she wipes them. "Your story is forever chiseled on our stones."`
+			`	She turns and leaves you there, holding a Tear of Skade, and shivering in Nifel's unforgiving cold.`
+
+
+
+mission "The Book of Skadenga"
+	invisible
+	source
+		government "Republic"
+	to offer
+		random < 10
+		or
+			has "Stones of Skadenga 4: done"
+			and
+				has "Skadenga Call 2: done"
+				has "event: the book"
+	on offer
+		conversation
+		`As you wander through the spaceport, something catches your eye at a kiosk selling gifts, trinkets, and other odds and ends.`
+		`	There's an old-style book there, bound up in a ice-blue cover, titled: "The Skakdenga Saga: Against All Odds, by Hjlod Hrimnirsdottir."`
+		choice
+			`	(Ignore the book and move on.)`
+				goto ignore
+			`	(Pick the book up.)`
+			branch "windblain"
+				has "Stones of Skadenga 4: done"
+			`	The book feels solid in your hands, and you scan through it. There are chapters about near death survival on Nifel, the subtle hostility of Asgard, the sudden death of the old Ondurdis, who you learn was named Eymani, and the triumph of Hjlod and the survivors on Nifel.`
+				goto end
+			label windblain
+			`	The book feels solid in your hands, and you scan through it. There are chapters about near death survival on Nifel, corrupt Deep officials, the colorful haulers on Muspel, and the ultimate triumph on Windblain.`
+			label end
+			`	You place the book back on its stand as the attendant notices you. "It's pretty popular, that one. I hear it started this whole 'back-to-nature' primalism cult." He shrugs. "Whatever sells, ya know. Anyways, you buying?"`
+			`	You decline. You already know that story.`
+				decline
+			label ignore
+			`	You keep walking. You already know that story.`
+				decline
+
+
+
+mission "The Angry Letter"
```

Comment:
This feels like setup for a future mission chain. If that's the case, I'd suggest removing it from this PR and including it with that mission chain whenever it's PRed.

Reply:
Removed it.